### ISSUE TITLE
adding bindings to HtmlVideoElement

### ISSUE THREAD

**Issue by BlueHotDog** [reasonml-community/bs-webapi-incubator#173]
Hi, I want to add bindings to HtmlVideoElement(defined here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLVideoElement). seems like there's a missing bindings for HtmlMediaElement(also the type is missing on Ocaml Dom).
Should i take a stab at it?
If so, how would i do that without the relevant ocaml type?

**Comment by glennsl** [comment]
Yeah, that would be great. You can just use a local `type t` for now, as well as submit a PR to `bs-platform` to add it to `Dom`. Once that's been merged and released we can link the local `t` type to that, and it won't even be a breaking change (other than requiring an updated bs-platform).

**Comment by illusionalsagacity** [comment]
I have a [PR up](https://github.com/rescript-lang/rescript-compiler/pull/4622) for rescript-compiler to add the phantom type to Dom. @BlueHotDog are you working on this? If not I can

**Comment by BlueHotDog** [comment]
hey, i'm not working on this :) feel free to PR :)
